netci.groovy

```diff
@@ -831,9 +835,17 @@ combinedScenarios.each { scenario ->
                                         // Get corefx
                                         buildCommands += "git clone https://github.com/dotnet/corefx fx"
                                         
+                                        // Set environment variable
+                                        def envVars = Constants.jitStressModeScenarios[scenario]
+                                        envVars.each{ VarName, Value   ->
+                                            if (VarName != '') {
+                                                buildCommands += "export ${VarName}=${Value}"
```

Comment:
Oh.. right. Thanks for finding this. I updated the changes to set environment variable in the same command line as build.sh.